docs/usage/configuration-options.md

```diff
@@ -3385,6 +3385,22 @@ In case there is a need to configure them manually, it can be done using this `r
 
 The field supports multiple URLs but it is datasource-dependent on whether only the first is used or multiple.
 
+## renovateCompatibility
+
+This capability exists so that Renovate maintainers can change default behavior without forcing all existing users to adopt the new behavior immediately.
```

Comment:
```suggestion
We (the Renovate maintainers) created the `renovateCompatibility` config so that we can change Renovate's default behavior, without forcing users to use the new behavior immediately.
```

Reply:
I think I'd prefer it to be less personal/more objective, happy to remove references to maintainers altogether

Reply:
Let's remove the reference to maintainers, it's not that interesting for Renovate end-users. Users only care about what the feature does for them. :wink:

Reply:
```suggestion
```